src/components/hide-this-page/index.md.njk

```diff
@@ -0,0 +1,71 @@
+---
+title: Hide this page
+description: Help users quickly hide the current page
+section: Components
+backlog_issue_id: 213
+layout: layout-pane.njk
+---
+
+{% from "_example.njk" import example %}
+
+Help users to exit a service quickly and safely.
+
+{{ example({group: "components", item: "hide-this-page", example: "default", html: true, nunjucks: true, open: false, size: "s"}) }}
+
+## When to use this component
+
+Use the hide this page component when some people could be put in danger or at risk of harm by using your service.
```

Comment:
```suggestion
Use the hide this page component when some people could be put in danger or at risk of harm by using your service.
```

From Hannah Jump:
I think this kind of sentence is what needs to be clearer:
> Use the hide this page component when some people could be put in danger or at risk of harm by using your service. For example, a service about domestic abuse.


I think:
- the emphasis needs to be made clear that someone is not put at harm by using the service, but the harm comes when a perpetrator sees or knows someone has used the service. The perpetrator is the person causing the harm, not the service itself. This is very important, and in line with ways charities etc talk about domestic abuse - the perpetrator is solely to blame)
- the information on the service might not seem sensitive, but it could be if it suggests someone is going to do something - like flee or be independent. I think it is important to acknowledge it is contextual.